README.md

```diff
@@ -1833,3 +1834,4 @@ support for C.
 [619]: https://projects.malikania.fr/bcc
 [620]: https://github.com/pspdev/pspsdk
 [621]: https://github.com/pspdev/pspsdk/blob/master/LICENSE
+[622]: https://github.com/Kode
```

Comment:
```suggestion
[622]: https://github.com/Kode/Kha
```

I think you should link to a project repo, not a GitHub organization.

@oz123 This repo that I've suggested in the Kobe org, it has recent commits, over 70 contributors, and 1.3K stars.

Reply:
In my suggestion, I changed the repo link to the dedicated site.